src/Model/Casts/CastResolver.php

```diff
@@ -11,11 +11,17 @@ class CastResolver
     private const DATE_TIME = 'datetime';
     private const IMMUTABLE_DATE_TIME = 'immutable_datetime';
 
+    /**
+     * @var array|string[]
```

Comment:
```suggestion
     * @var string[]
```
I think only `string[]` is enough here.

Reply:
Alright, I changed here.